website/tailwind.config.js

```diff
@@ -6,7 +6,7 @@ module.exports = {
   theme: {
     extend: {
       backgroundImage: {
-        'hero-pattern': "url('/img/gradients.png')",
+        'hero-pattern': 'url(\'/img/gradients.png\')',
```

Comment:
you need to remove that change for now for the linter check

Reply:
done